scripts/pythondistdeps.py

```diff
@@ -197,6 +200,13 @@ def normalize_name(name):
     # At least one type of normalization must be provided
     assert normalized_names_provide_pep503 or normalized_names_provide_legacy
 
+    # Is this script being run for an extras subpackage?
+    extras_subpackage = None
+    if args.package_name:
+        package_name_parts = args.package_name.partition('+')
+        if package_name_parts[2]:
+            extras_subpackage = package_name_parts[2]
```

Comment:
Maybe?

```suggestion
    if args.package_name:
        package_name_parts = args.package_name.partition('+')
        extras_subpackage = package_name_parts[2] or None
```

Reply:
Beautiful!

Reply:
Oh, I didn't see you also removed the `extras_subpackage = None` on top. That sadly breaks, because when checking `if extras_subpackage` it throws `NameError: name 'extras_subpackage' is not defined`

Reply:
Sorry. Let me redo this part in a suggestion.